src/client/mesh_generator_thread.cpp

```diff
@@ -231,6 +231,15 @@ void MeshUpdateQueue::cleanupCache()
 	g_profiler->avg("MeshUpdateQueue MapBlock cache size kB",
 			mapblock_kB * m_cache.size());
 
+	// Iterating the entire cache can get pretty expensive so don't do it too often
```

Comment:
Might make sense to put this before the profiler call, unless we want to know how many times addBlock is called in 3 seconds.

Reply:
The idea was that the profiler still reflects cache size changes immediately, doesn't matter that much tho.